src/wizards/generateProject/generationWizard.ts

```diff
@@ -139,7 +139,26 @@ export async function generateProjectWizard() {
       prompt: 'Your resource name',
       validate: validateResourceName
     });
-    return (input: MultiStepInput) => ExtensionsPicker.createExtensionsPicker(input, state, { showLastUsed: true, showRequiredExtensions: true, allowZeroExtensions: true });
+    return (input: MultiStepInput) => ExtensionsPicker.createExtensionsPicker(
+      input, state, { showLastUsed: true, showRequiredExtensions: true, allowZeroExtensions: true },
+      inputGenerateSampleCode);
+  }
+
+  async function inputGenerateSampleCode(input: MultiStepInput, state: Partial<ProjectGenState>) {
```

Comment:
I should make this step conditional on if the API supports not generating sample code by looking at the openapi. code.quarkus.io supports exclusing examples, but code.quarkus.redhat.com doesn't